bedrock/firefox/urls.py

```diff
@@ -73,7 +73,7 @@
     # This dummy page definition makes it possible to link to /firefox/ (Bug 878068)
     url('^firefox/$', views.fx_home_redirect, name='firefox'),
 
-    url('^firefox/os/$', views.firefox_os_index, name='firefox.os.index'),
+    url('^firefox/os/$', views.fxos, name="fxos"),
```

Comment:
I'm guessing this should be single quotes for consistency. Also is there a reason to change the name from `firefox.os.index`? Maybe @pmclanahan can advise here.